packages/eslint-config/index.js

```diff
@@ -46,9 +46,9 @@ module.exports = {
     requireConfigFile: false,
     babelOptions: getProjectBabelOptions(),
   },
-  rules: {
-    '@redwoodjs/unsupported-route-components': 'error',
-  },
+  // rules: {
```

Comment:
This should not be commented out but was needed due to https://github.com/redwoodjs/redwood/issues/8792